docs/sources/enterprise/license-expiration.md

```diff
@@ -0,0 +1,54 @@
++++
+title = "License Expiration"
+description = ""
+keywords = ["grafana", "licensing"]
+type = "docs"
+[menu.docs]
+parent = "enterprise"
+weight = 8
++++
+
+# License expiration 
+
+If your license has expired most of Grafana keeps working as normal. Some enterprise functionality stops or runs with reduced functionality and Grafana displays a banner informing the users that Grafana is running on an expired license. Your Grafana admin needs to upload a new license file to restore full functionality.
+
+> Replace your license as soon as possible. Running Grafana Enterprise with an expired license is unsupported and can lead to unexpected consequences.
+
+## Replacing your license
+
+1. Locate your current `license.jwt` file. In a standard installation it is stored inside Grafana's data directory, which on a typical Linux installation is in `/var/lib/grafana/data`. This location might be overridden in the ini file [Configuration](https://grafana.com/docs/grafana/latest/installation/configuration/)
+```
+[enterprise]
+license_path = /path/to/your/license.jwt
+```
+The configuration file's location may also be overridden by the `GF_ENTERPRISE_LICENSE_PATH` environment variable.
+2. Log in to your [Grafana.com](https://grafana.com/login) user and make sure you're in the correct organization in the dropdown at the top of the page.
+3. Under the **Grafana Enterprise** section in the menu bar to the left, choose licenses and download the currently valid license with which you want to run Grafana. If you cannot see a valid license on Grafana.com, please contact your account manager at Grafana Labs to renew your subscription.
+4. Replace the current `license.jwt`-file with the one you've just downloaded.
+5. Restart Grafana.
+
+## Data source permissions
+
+Your current data source permissions will keep working as expected, but you'll be unable to add new data source permissions until the license has been renewed.
+
+## Reporting
+
+- You're unable to configure new reports or generate previews.
+- Scheduled reports are not generated or sent.
+
+## SAML authentication
+
+SAML is not affected by an expired license.
+
+## LDAP authentication
+
+- LDAP synchronization is not affected by an expired license.
+- Enhanced LDAP debugging is unavailable.
+
+## Enterprise plugins
+
+Enterprise plugins might stop working.
```

Comment:
Might stop or will stop?

Reply:
Currently it depends on the plugin how it handles licenses, so might. Not sure what we'll do about that long-term.

Reply:
I don't think any of them will stop working right now. There are no license checks afaik.

Reply:
Um, do we really want to mention that? And is there an issue open for this?

Reply:
For this PR, though, seems that the current wording is the best we can do given current functionality.

Reply:
Agreed, let's merge it as is.